config/rbac/role.yaml

```diff
@@ -110,6 +110,18 @@ rules:
   - patch
   - update
   - watch
+- apiGroups:
+  - kueue.x-k8s.io
+  resources:
+  - clusterqueues
+  verbs:
+  - create
```

Comment:
least privilege... does the controller need create and delete?

Reply:
not needed and removed